ibm/mas_devops/roles/suite_manage_load_dbc_scripts/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
would you be able to add an extra interna var, something as:
```
mas_app_id_supported:
  - 'manage'
  - 'health'
```

and then add an assertion right at the beginning of the `tasks/main.yml` that validates that the passed `mas_app_id` matches one of the `mas_app_id_supported`?

Here's a sample of how you'd do it:
https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/cp4d/tasks/main.yml#L13

Reply:
Added and tested
<img width="1074" alt="image" src="https://github.com/ibm-mas/ansible-devops/assets/12377774/183ae91e-b1e0-4c51-93ec-50165657b28b">